docs/tutorial/t-setup-environment.md

```diff
@@ -1,7 +1,9 @@
 ## Setup Kubernetes and Spark Client Snap
 
-In order to be able to work with Kubernetes,  first we first to set up a Kubernetes cluster. In this tutorial we use [MicroK8s](https://microk8s.io/), 
-which is the simplest production-grade conformant K8s.
+In order to be able to work with Kubernetes, we first need to set up a Kubernetes cluster. 
+
+In this tutorial we use [MicroK8s](https://microk8s.io/), which is the simplest production-grade conformant K8s.
+If you would like to run this tutorial on other K8s installation, please refer to the "How-to setup on K8s" user guide. 
```

Comment:
Can you link the `How to setup K8s` here?

Reply:
Yes, I actually wanted to first have a first round of review, then when we are close to finalizing, I will create the topic in Discourse and add the link appropriately

Reply:
Done!